sheepCount.js

```diff
@@ -38,5 +38,69 @@ console.log(sheepCount(11)) //110
 */
 
 function sheepCount(N) {
-	//your code
+
+	//create containers for status of each number
+	var zero = false;
+	var one = false;
+	var two = false;
+	var three = false;
+	var four = false;
+	var five = false;
+	var six = false;
+	var seven = false;
+	var eight = false;
+	var nine = false;
+	//if n=0, return "INSOMNIA"
+	if (N <== 0) {
+		return "INSOMNIA";
+	}
+	// contianing loop for sequential products
+	for (i = 1; i< 100 ; i++) {
```

Comment:
using a while loop is better because you do not know how many times you need to iterate to see all the digits.